.github/workflows/flowzone.yml

```diff
@@ -300,37 +300,25 @@ jobs:
         working-directory: .
         shell: bash
 
-    outputs:
-      versionbot: ${{ steps.versionbot_checkout.outcome }}
-
     steps:
-      - name: Checkout versioned source
-        id: versionbot_checkout
-        continue-on-error: true
-        uses: actions/checkout@v3
-        with:
-          ref: versionbot/pr/${{ github.event.pull_request.number }}
-          fetch-depth: 2
-          token: ${{ secrets.FLOWZONE_TOKEN }}
-
-      - name: Assert that versioned source is up-to-date
-        if: ${{ steps.versionbot_checkout.outcome == 'success' }}
-        run: git merge-base --is-ancestor ${{ github.event.pull_request.head.sha }} HEAD
-
-      # Fallback to unversioned source if versionbot branch does not exist
       - name: Checkout original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         uses: actions/checkout@v3
         with:
-          fetch-depth: 1
+          fetch-depth: 0
           token: ${{ secrets.FLOWZONE_TOKEN }}
 
-      # Run balena-versionist in Github actions when the versionbot branch could not be checked out
       - name: Version original source
-        if: ${{ steps.versionbot_checkout.outcome == 'failure' }}
         run: |
-          npm install -g balena-versionist versionist
-          balena-versionist
+          if [ -f repo.yml ]
```

Comment:
So we will require repo.yml for automatic versioning? Is this instead of turning versioning on/off with an input?

Reply:
This is just one of the checks since we can skip versioning if the file does not exist.  We have `ignore_versionbot_branch` and `skip_versioning` to bypass versioning.